include/unifex/let_error.hpp

```diff
@@ -208,19 +220,20 @@ class _frcvr<Source, Error, Receiver>::type {
   operation* op_;
 };
 
-template <typename Source, typename Error, typename Receiver>
-class _op<Source, Error, Receiver>::type {
-  using source_receiver = receiver_type<Source, Error, Receiver>;
-  using final_receiver = final_receiver_type<Source, Error, Receiver>;
+template <typename Source, typename Func, typename Receiver>
+class _op<Source, Func, Receiver>::type {
+  using source_receiver = receiver_type<Source, Func, Receiver>;
+  using final_receiver = final_receiver_type<Source, Func, Receiver>;
 
 public:
-  template <typename Error2, typename Receiver2>
-  explicit type(Source&& source, Error2&& error, Receiver2&& dest)
+  template <typename Func2, typename Receiver2>
+  explicit type(Source&& source, Func2&& error, Receiver2&& dest)
```

Comment:
Can we change the parameter name from `error` to `func` here to match the template parameter name?
And maybe also the member name?